bundles/org.openhab.ui/web/src/components/widgets/generic-widget-component.vue

```diff
@@ -11,10 +11,10 @@
   <generic-widget-component v-else-if="componentType && componentType.startsWith('widget:') && visible" :context="childWidgetContext()" @command="onCommand" />
   <component v-else-if="componentType && componentType.startsWith('oh-') && visible" :is="componentType" :context="context" @command="onCommand" />
   <div v-else-if="componentType && componentType === 'Label' && visible" :class="config.class" :style="config.style">
-    {{ config.text }}
+    {{ (config.text) ? config.text : 'undefined' }}
```

Comment:
At first sight this would transform all [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) expressions (i.e. also `false`, `0`, `null`, empty strings etc.) into `undefined`  which is surely not what we want.

Honestly I don't understand what #1696 was supposed to be fixing, #1026 was about the widget expression tester so that's probably where a fix is needed, no need to make global changes to handle this case I believe.

Reply:
Ooops, you are right.

#1696 tried to fix #1026, but it was indeed the wrong way. I've just rechecked the widget expression tester, it is possible to make the required change in the developer-sidebar. I'll fix that.